### ISSUE TITLE
AEN-BERT Twitter Analysis Accuracy

### ISSUE THREAD

**Issue by Sanjaytp** [songyouwei/ABSA-PyTorch#92]
Hi, I am using the AEN-BERT model to do a 'Sentiment Analysis' on Twitter data.

I first used the default hyper-parameter values along with the Train and Test data provided in this repository, but could reach a Validation Accuracy and Test Accuracy of only around 74%.

I then tried different 'learning rates' suggested in the code itself and also different batch sizes.
I also tried to use the Adamax optimizer too with different 'learning rates' (both default and other learning rates suggested in the code), but still not able to go beyond 74%.

Is there anything else I can try? May be increase the size of the dataset? Or, is there something else I am missing..?

_Sanjay
